src/components/HeaderMobile/index.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+
+import {
+  Drawer,
+  DrawerBody,
+  DrawerContent,
+  DrawerFooter,
+  DrawerHeader,
+  DrawerOverlay,
+  IconButton,
+  useDisclosure,
+} from '@chakra-ui/react'
+import { FaBars } from 'react-icons/fa'
+
+import { HeaderNav, HeaderTop } from '@components'
+
+type HeaderMobileProps = {
+  hasScroll?: boolean
+  isScrolled?: boolean
+}
+
+export const HeaderMobile = (props: HeaderMobileProps) => {
+  const { isOpen, onToggle, onClose } = useDisclosure()
+  const { hasScroll, isScrolled } = props
+
+  return (
+    <>
+      <IconButton
+        display={{ base: 'flex', lg: 'none' }}
+        variant="outline"
+        colorScheme={
+          !hasScroll ? 'blackAlpha' : isScrolled ? 'blackAlpha' : 'whiteAlpha'
+        }
+        onClick={onToggle}
+        aria-label="menu"
+        icon={<FaBars />}
+      />
+      <Drawer isOpen={isOpen} onClose={onClose}>
+        <DrawerOverlay />
+        <DrawerContent>
+          <DrawerHeader>Menu</DrawerHeader>
+          <DrawerBody>
+            <HeaderNav direction="column" />
+          </DrawerBody>
+          <DrawerFooter justifyContent="center">
+            <HeaderTop />
```

Comment:
`HeaderTop` component looks here weird. The name of the component might be changed as `LocaleSwitcher`